utils/phpstan-baseline/codeigniter.unknownServiceMethod.neon

```diff
@@ -0,0 +1,33 @@
+# total 6 errors
```

Comment:
If the text _# total 6 errors_ needs to be updated every time an error is fixed in the future, it could indeed make maintenance harder, especially if the number of errors is subject to frequent changes.

Reply:
I'm not sure what you mean. Whenever you fix an error, you just need to rerun `composer phpstan:baseline` and that would take care updating all baselines affected

Reply:
Oh, I see! So, the _# total 6 errors_ baseline file automatically updates after running the command.

Reply:
Yes, the updating of the comment is part of the split command